docs/Documentation/QuickStart.md

```diff
@@ -55,306 +56,142 @@ IoTDB provides you two installation methods, you can refer to the following sugg
-IoTDB> SHOW TIMESERIES
-===  Timeseries Tree  ===
-
-{
-	"root":{
-		"ln":{
-			"wf01":{
-				"wt01":{
-					"temperature":{
-						"args":"{}",
-						"StorageGroup":"root.ln",
-						"DataType":"FLOAT",
-						"Compressor":"UNCOMPRESSED",
-						"Encoding":"RLE"
-					},
-					"status":{
-						"args":"{}",
-						"StorageGroup":"root.ln",
-						"DataType":"BOOLEAN",
-						"Compressor":"UNCOMPRESSED",
-						"Encoding":"PLAIN"
-					}
-				}
-			}
-		}
-	}
-}
+SET STORAGE GROUP TO root.fit.d1;
+SET STORAGE GROUP TO root.fit.d2;
+SET STORAGE GROUP TO root.fit.p;
+CREATE TIMESERIES root.fit.d1.s1 WITH DATATYPE=INT32,ENCODING=RLE;
+CREATE TIMESERIES root.fit.d1.s2 WITH DATATYPE=TEXT,ENCODING=PLAIN;
+CREATE TIMESERIES root.fit.d2.s1 WITH DATATYPE=INT32,ENCODING=RLE;
+CREATE TIMESERIES root.fit.d2.s3 WITH DATATYPE=INT32,ENCODING=RLE;
+CREATE TIMESERIES root.fit.p.s1 WITH DATATYPE=INT32,ENCODING=RLE;
 ```
-Now, for your conveniect, SHOW TIMESERIES clause also supports extention syntax, the pattern is (for further details, check Chapter 5 of the User Guide):
 
+### Run import shell
 ```
-SHOW TIMESERIES <PATH>
-```
-
-Here is the example:
+# Unix/OS X
+> tools/import-csv.sh -h <ip> -p <port> -u <username> -pw <password> -f <xxx.csv>
 
+# Windows
+> tools\import-csv.bat -h <ip> -p <port> -u <username> -pw <password> -f <xxx.csv>
 ```
-IoTDB> SHOW TIMESERIES root.ln.wf01.wt01
-+------------------------------+--------------+--------+--------+
-|                    Timeseries| Storage Group|DataType|Encoding|
-+------------------------------+--------------+--------+--------+
-|      root.ln.wf01.wt01.status|       root.ln| BOOLEAN|   PLAIN|
-| root.ln.wf01.wt01.temperature|       root.ln|   FLOAT|     RLE|
-+------------------------------+--------------+--------+--------+
-Total timeseries number = 2
-Execute successfully.
-```
-We can also use SHOW STORAGE GROUP to check created storage group:
 
-```
-IoTDB> show storage group
-+-----------------------------------+
-|                      Storage Group|
-+-----------------------------------+
-|                            root.ln|
-+-----------------------------------+
-Total storage group number = 1
-Execute successfully.
-It costs 0.006s
-```
+### Error data file
 
+`csvInsertError.error`
 
-Insert timeseries data is the basic operation of IoTDB, you can use 'INSERT' command to finish this:
+# Usage of export-csv.sh
 
+### Run export shell
 ```
-IoTDB> INSERT INTO root.ln.wf01.wt01(timestamp,status) values(100,true);
-execute successfully.
-IoTDB> INSERT INTO root.ln.wf01.wt01(timestamp,status,temperature) values(200,false,20.71) execute successfully.
-```
-The data we've just inserted displays like this:
+# Unix/OS X
+> tools/export-csv.sh -h <ip> -p <port> -u <username> -pw <password> -td <xxx.csv> [-tf <time-format>]
 
+# Windows
+> tools\export-csv.bat -h <ip> -p <port> -u <username> -pw <password> -td <xxx.csv> [-tf <time-format>]
 ```
```

Comment:
I think it is better to explain the parameters, especially the last one. Or you may link to the detailed user guide.

Reply:
removed